react-admin/src/pages/auth/Login.jsx

```diff
@@ -65,6 +65,7 @@ function Login() {
                 name="email"
                 autoFocus={true}
                 register={register("email")}
+                errors={errors}
```

Comment:
this errors can be backend and frontend both. basically method `getErrorMessage` needs to check for either frontend and backend then return the error appropriate.